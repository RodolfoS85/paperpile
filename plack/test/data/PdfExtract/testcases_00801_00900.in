---
file: ../../../PDFREPOSITORY/000801.pdf
---
file: ../../../PDFREPOSITORY/000802.pdf
---
file: ../../../PDFREPOSITORY/000803.pdf
---
file: ../../../PDFREPOSITORY/000804.pdf
---
file: ../../../PDFREPOSITORY/000805.pdf
---
file: ../../../PDFREPOSITORY/000806.pdf
---
file: ../../../PDFREPOSITORY/000807.pdf
---
file: ../../../PDFREPOSITORY/000808.pdf
---
file: ../../../PDFREPOSITORY/000809.pdf
---
file: ../../../PDFREPOSITORY/000810.pdf
---
file: ../../../PDFREPOSITORY/000811.pdf
---
file: ../../../PDFREPOSITORY/000812.pdf
---
file: ../../../PDFREPOSITORY/000813.pdf
---
file: ../../../PDFREPOSITORY/000814.pdf
---
file: ../../../PDFREPOSITORY/000815.pdf
---
file: ../../../PDFREPOSITORY/000816.pdf
---
file: ../../../PDFREPOSITORY/000817.pdf
---
file: ../../../PDFREPOSITORY/000818.pdf
---
file: ../../../PDFREPOSITORY/000819.pdf
---
file: ../../../PDFREPOSITORY/000820.pdf
---
file: ../../../PDFREPOSITORY/000821.pdf
---
file: ../../../PDFREPOSITORY/000822.pdf
---
file: ../../../PDFREPOSITORY/000823.pdf
---
file: ../../../PDFREPOSITORY/000824.pdf
---
file: ../../../PDFREPOSITORY/000825.pdf
---
file: ../../../PDFREPOSITORY/000826.pdf
---
file: ../../../PDFREPOSITORY/000827.pdf
---
file: ../../../PDFREPOSITORY/000828.pdf
---
file: ../../../PDFREPOSITORY/000829.pdf
---
file: ../../../PDFREPOSITORY/000830.pdf
---
file: ../../../PDFREPOSITORY/000831.pdf
---
file: ../../../PDFREPOSITORY/000832.pdf
---
file: ../../../PDFREPOSITORY/000833.pdf
---
file: ../../../PDFREPOSITORY/000834.pdf
---
file: ../../../PDFREPOSITORY/000835.pdf
---
file: ../../../PDFREPOSITORY/000836.pdf
---
file: ../../../PDFREPOSITORY/000837.pdf
---
file: ../../../PDFREPOSITORY/000838.pdf
---
file: ../../../PDFREPOSITORY/000839.pdf
---
file: ../../../PDFREPOSITORY/000840.pdf
---
file: ../../../PDFREPOSITORY/000841.pdf
---
file: ../../../PDFREPOSITORY/000842.pdf
---
file: ../../../PDFREPOSITORY/000843.pdf
---
file: ../../../PDFREPOSITORY/000844.pdf
---
file: ../../../PDFREPOSITORY/000845.pdf
---
file: ../../../PDFREPOSITORY/000846.pdf
---
file: ../../../PDFREPOSITORY/000847.pdf
---
file: ../../../PDFREPOSITORY/000848.pdf
---
file: ../../../PDFREPOSITORY/000849.pdf
---
file: ../../../PDFREPOSITORY/000850.pdf
---
file: ../../../PDFREPOSITORY/000851.pdf
---
file: ../../../PDFREPOSITORY/000852.pdf
---
file: ../../../PDFREPOSITORY/000853.pdf
---
file: ../../../PDFREPOSITORY/000854.pdf
---
file: ../../../PDFREPOSITORY/000855.pdf
---
file: ../../../PDFREPOSITORY/000856.pdf
---
file: ../../../PDFREPOSITORY/000857.pdf
---
file: ../../../PDFREPOSITORY/000858.pdf
---
file: ../../../PDFREPOSITORY/000859.pdf
---
file: ../../../PDFREPOSITORY/000860.pdf
---
file: ../../../PDFREPOSITORY/000861.pdf
---
file: ../../../PDFREPOSITORY/000862.pdf
---
file: ../../../PDFREPOSITORY/000863.pdf
---
file: ../../../PDFREPOSITORY/000864.pdf
---
file: ../../../PDFREPOSITORY/000865.pdf
---
file: ../../../PDFREPOSITORY/000866.pdf
---
file: ../../../PDFREPOSITORY/000867.pdf
---
file: ../../../PDFREPOSITORY/000868.pdf
---
file: ../../../PDFREPOSITORY/000869.pdf
---
file: ../../../PDFREPOSITORY/000870.pdf
---
file: ../../../PDFREPOSITORY/000871.pdf
---
file: ../../../PDFREPOSITORY/000872.pdf
---
file: ../../../PDFREPOSITORY/000873.pdf
---
file: ../../../PDFREPOSITORY/000874.pdf
---
file: ../../../PDFREPOSITORY/000875.pdf
---
file: ../../../PDFREPOSITORY/000876.pdf
---
file: ../../../PDFREPOSITORY/000877.pdf
---
file: ../../../PDFREPOSITORY/000878.pdf
---
file: ../../../PDFREPOSITORY/000879.pdf
---
file: ../../../PDFREPOSITORY/000880.pdf
---
file: ../../../PDFREPOSITORY/000881.pdf
---
file: ../../../PDFREPOSITORY/000882.pdf
---
file: ../../../PDFREPOSITORY/000883.pdf
---
file: ../../../PDFREPOSITORY/000884.pdf
---
file: ../../../PDFREPOSITORY/000885.pdf
---
file: ../../../PDFREPOSITORY/000886.pdf
---
file: ../../../PDFREPOSITORY/000887.pdf
---
file: ../../../PDFREPOSITORY/000888.pdf
---
file: ../../../PDFREPOSITORY/000889.pdf
---
file: ../../../PDFREPOSITORY/000890.pdf
---
file: ../../../PDFREPOSITORY/000891.pdf
---
file: ../../../PDFREPOSITORY/000892.pdf
---
file: ../../../PDFREPOSITORY/000893.pdf
---
file: ../../../PDFREPOSITORY/000894.pdf
---
file: ../../../PDFREPOSITORY/000895.pdf
---
file: ../../../PDFREPOSITORY/000896.pdf
---
file: ../../../PDFREPOSITORY/000897.pdf
---
file: ../../../PDFREPOSITORY/000898.pdf
---
file: ../../../PDFREPOSITORY/000899.pdf
---
file: ../../../PDFREPOSITORY/000900.pdf
