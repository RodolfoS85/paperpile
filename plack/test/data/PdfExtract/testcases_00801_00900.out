---
authors: 'Lindahl, Maria and Timmusk, Tonis and Rossi, Jari and Saarma, Mart and Airaksinen, Matti S.'
doi: 10.1006/mcne.2000.0845
title: Expression and Alternative Splicing of Mouse Gfra4 Suggest Roles in Endocrine Cell Development
---
authors: 'Lindow, Morten and Gorodkin, Jan'
doi: 10.1089/dna.2006.0551
title: Principles and Limitations of Computational MicroRNA Gene and Target Finding
---
authors: 'Linhart, Chaim and Halperin, Yonit and Shamir, Ron'
doi: 10.1101/gr.076117.108
title: 'Transcription factor and microRNA motif discovery: The Amadeus platform and a compendium of metazoan target sets'
---
authors: 'Link, Todd M. and Valentin-Hansen, Poul and Brennan, Richard G.'
doi: 10.1073/pnas.0908744106
title: Structure of Escherichia coli Hfq bound to polyriboadenylate RNA
---
authors: 'Linke, Katrin and Wolfram, Tobias and Bussemer, Johanna and Jakob, Ursula'
doi: 10.1074/jbc.M307491200
title: The Roles of the Two Zinc Binding Sites in DnaJ
---
authors: 'Liu, Bing and Xia, Yiyuan and Yu, Philip S.'
title: Clustering Through Decision Tree Construction
---
authors: 'Liu, Chunmei and Song, Yinglei and Shapiro, Louis'
title: 'RNA Folding Including Pseudoknots: A New Parameterized Algorithm and Improved Upper Bound'
---
authors: 'Liu, Huiqing and Han, Hao and Li, Jinyan'
title: An In-Silico Method for Prediction of Polyadenylation Signals in Human Sequences
---
authors: 'Liu, X.'
title: 'BIOPROSPECTOR: DISCOVERING CONSERVED DNA MOTIFS IN UPSTREAM REGULATORY REGIONS OF CO-EXPRESSED GENES'
---
authors: 'Liu, HONG-XIANG and Chew, Shern L. and Cartegni, Luca and Zhang, Michael Q. and Krainer, Adrian R.'
title: Exonic Splicing Enhancer Motif Recognized by Human SC35 under Splicing Conditions
---
authors: 'Liu, Ying'
doi: 10.1021/ci049875d
title: A Comparative Study on Feature Selection Methods for Drug Discovery
---
authors: 'Liu, Yang and Engelman, Donald M and Gerstein, Mark'
title: 'Genomic analysis of membrane protein families: abundance and conserved motifs'
---
authors: 'Liu, Wei and Li, Dong and Zhang, Jiyang and Zhu, Yunping and He, Fuchu'
doi: 10.1186/1471-2105-7-515
title: 'SigFlux: A novel network feature to evaluate the importance of proteins in signal transduction networks'
---
authors: 'Liu, Fei and Tong, Huan and Ou-Yang, Zhong-can'
doi: 10.1529/biophysj.105.070540
title: Force Unfolding Single RNAs
---
authors: 'Liu, Jianghui and Wang, Jason TL and Hu, Jun and Tian, Bin'
doi: 10.1186/1471-2105-6-89
title: A method for aligning RNA secondary structures and its application to RNA motif detection
---
authors: 'Liu, Hong-Xiang and Zhang, Michael and Krainer, Adrian R.'
title: Identification of functional exonic splicing enhancer motifs recognized by individual SR proteins
---
authors: 'Livny, Jonathan and Teonadi, Hidayat and Livny, Miron and Waldor, Matthew K.'
doi: 10.1371/journal.pone.0003197
title: 'High-Throughput, Kingdom-Wide Prediction and Annotation of Bacterial Non-Coding RNAs'
---
authors: 'Lo Conte, Loredana and Ailey, Bart and Hubbard, Tim J. P. and Brenner, Steven E. and Murzin, Alexey G. and Chothia, Cyrus'
title: 'SCOP: a Structural Classification of Proteins database'
---
authors: 'Lohmann, Jan U. and Hong, Ray L. and Hobe, Martin and Busch, Maximilian A. and Parcy, Francois and Simon, Rudiger and Weigel, Detlef'
title: A Molecular Link between Stem Cell Regulation and Floral Patterning in Arabidopsis
---
authors: 'Long, Manyuan and Deutsch, Michael'
title: Association of Intron Phases with Conservation at Splice Site Sequences and Evolution of Spliceosomal Introns
---
authors: 'Long, Dang and Lee, Rosalind and Williams, Peter and Chan, Chi Yu and Ambros, Victor and Ding, Ye'
doi: 10.1038/nsmb1226
title: Potent effect of target structure on microRNA function
---
authors: 'Lopez-Bigas, Nuria and Audit, Benjamin and Ouzounis, Christos and Parra, Genis and Guigo, Roderic'
doi: 10.1016/j.febslet.2005.02.047
title: Are splicing mutations the most frequent cause of hereditary disease?
---
authors: 'LÓPEZ DE QUINTO, Sonia and Lafuente, Esther and MARTÍNEZ-SALAS, ENCARNACIÓN'
title: 'IRES interaction with translation initiation factors: Functional characterization of novel RNA contacts with eIF3, eIF4B, and eIF4GII'
---
authors: 'Lopez de Silanes, Isabel and Zhan, Ming and Lal, Ashish and Yang, Xiaoling and Gorospe, Myriam'
doi: 10.1073/pnas.0306453101
title: Identification of a target RNA motif for RNA-binding protein HuR
---
authors: 'Loraine, Ann E. and Helt, Gregg A. and Cline, Melissa S. and SIANI-ROSE, Michael A.'
title: EXPLORING ALTERNATIVE TRANSCRIPT STRUCTURE IN THE HUMAN GENOME USING BLOCKS AND INTERPRO
---
authors: 'Loraine, Ann E and Helt, Gregg A'
title: 'Visualizing the genome: techniques for presenting human genome data and annotations'
---
authors: 'Lorenz, W. A. and Ponty, Y. and Clote, P.'
title: ASYMPTOTICS OF RNA SHAPES
---
authors: 'Lowe, Todd M. and Eddy, Sean R.'
title: A Computational Screen for Methylation Guide snoRNAs in Yeast
---
authors: 'Lowe, Todd M. and Eddy, Sean R.'
title: 'tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence'
---
authors: 'Landau, Gad M. and Parida, Laxmi and Weimann, Oren'
title: Gene Proximity Analysis Across Whole Genomes via PQ Trees
---
authors: 'Lu, Fan and Keles, Sunduz and Wright, Stephen J. and Wahba, Grace'
doi: 10.1073/pnas.0505411102
title: Framework for kernel regularization with application to protein clustering
---
authors: 'Lu, Yang and Turner, Robert J. and Switzer, Robert L.'
title: Function of RNA secondary structures in transcriptional attenuation of the Bacillus subtilis pyr operon
---
authors: 'Lück, Rupert and Gräf, Stefan and Steger, Gerhard'
title: 'ConStruct: a tool for thermodynamic controlled prediction of conserved secondary structure'
---
authors: 'Krippahl, Ludwig'
title: 'PSICO: Solving Protein Structures with Constraint Programming and Optimization'
---
authors: 'Lukashin, Alexander V. and Borodovsky, Mark'
title: 'GeneMark.hmm: new solutions for gene finding'
---
authors: 'Lukavsky, Peter J and Kim, Insil and Otto, Geoff A and Puglisi, Joseph D'
doi: 10.1038/nsb1004
title: Structure of HCV IRES domain II determined by NMR
---
authors: 'Lunde, Bradley M. and Moore, Claire and Varani, Gabriele'
doi: 10.1038/nrm2178
title: 'RNA-binding proteins: modular design for efficient function'
---
authors: 'Lunin, Vladimir Y. and Urzhumtsev, Alexandre and Bockmayr, Alexander'
title: Binary Integer Programming and its Use for Envelope Determination
---
authors: 'Luo, Daochun and Mari, Bernard and Stoll, Isabelle and Anglard, Patrick'
doi: 10.1074/jbc.M202494200
title: Alternative Splicing and Promoter Usage Generates an Intracellular Stromelysin 3 Isoform Directly Translated as an Active Matrix Metalloproteinase
---
authors: 'Lutz, Stefan and Benkovic, Stephen J'
title: Homology-independent protein engineering
---
authors: 'Lykke-Andersen, Søren and Jensen, Torben Heick'
title: 'CUT it out: silencing of noise in the transcriptome'
---
authors: 'Lykke-Andersen, Jens'
doi: 10.1038/sj.emboj.760015
title: Making structural sense of nonsense-mediated decay
---
authors: 'Lynch, Kristen W.'
doi: 10.1038/nri1497
title: CONSEQUENCES OF REGULATED PRE-mRNA SPLICING IN THE IMMUNE SYSTEM
---
authors: 'Lynch, Michael and Richardson, Aaron O'
title: The evolution of spliceosomal introns
---
authors: 'Lynch, Kristen W. and Weiss, Arthur'
doi: 10.1074/jbc.M102175200
title: A CD45 Polymorphism Associated with Multiple Sclerosis Disrupts an Exonic Splicing Silencer
---
authors: 'Lyngsø, Rune B. and Pedersen, Christian N. S.'
title: Pseudoknots in RNA Secondary Structures
---
authors: 'Lyngsø, Rune B. and Zuker, Michael and Pedersen, Christian N. S.'
title: An Improved Algorithm for RNA Secondary Structure Prediction
---
authors: 'Ma, Bin and Wang, Lusheng and Zhang, Kaizhong'
title: Computing similarity between RNA structures
---
authors: 'Maas, Stefan and Patt, Stephan and Schrey, Michael and Rich, Alexander'
doi: 10.1073/pnas.251531398
title: Underediting of glutamate receptor GluR-B mRNA in malignant gliomas
---
authors: 'Maas, Stefan and Rich, Alexander'
title: Changing genetic information through RNA editing
---
authors: 'Maayan, Avi and Jenkins, Sherry L. and Neves, Susana and Hasseldine, Anthony and Grace, Elizabeth and Dubin-Thaler, Benjamin and Eungdamrong, Narat J. and Weng, Gehzi and Ram, Prahlad T. and Jeremy Rice, J. and Kershenbaum, Aaron and Stolovitzky, Gustavo A. and Blitzer, Robert D. and Iyengar, Ravi'
title: Formation of Regulatory Patterns During Signal Propagation in a Mammalian Cellular Network
---
authors: 'Mabon, Stephen A. and Misteli, Tom'
doi: 10.1371/journal.pbio.0030374
title: Differential Recruitment of Pre-mRNA Splicing Factors to Alternatively Spliced Transcripts In Vivo
---
authors: 'MacDonald, David and Hunter, D L and Kelly, Karen and Jan, Naeem'
title: 'Self-avoiding walks in two to five dimensions: exact enumerations and series study'
---
authors: 'MacDonald, D and Joseph, S and Hunter, D L and Moseley, L L and Jan, N and Guttmann, A J'
title: Self-avoiding walks on the simple cubic lattice
---
authors: 'Macke, Thomas J. and Ecker, David J. and Gutell, Robin R. and Gautheret, Daniel and Case, David A. and Sampath, Rangarajan'
title: 'RNAMotif, an RNA secondary structure definition and search algorithm'
---
authors: 'Madras, Neal and Sokal, Alan D.'
title: 'Nonergodicity of Local, Length-Conserving Monte Carlo Algorithms for the Self-Avoiding Walk'
---
authors: 't, Neal Madras and Sokai, Alan D.'
title: 'The Pivot Algorithm: A Highly Efficient Monte Carlo Method for the Self-Avoiding Walk'
---
authors: 'Mäkinen, Veli'
title: Using Edit Distance in Point-Pattern Matching
---
authors: 'Maidak, Bonnie L. and Cole, James R. and Lilburn, Timothy G. and Parker, Jr., Charles T. and Saxman, Paul R. and Farris, Ryan J. and Garrity, George M. and Olsen, Gary J. and Schmidt, Thomas M. and Tiedje, James M.'
title: The RDP-II (Ribosomal Database Project)
---
authors: 'Majdalani, Nadim and Cunning, Christofer and Sledjeski, Darren and Elliott, Tom and Gottesman, Susan'
title: 'DsrA RNA regulates translation of RpoS message by an anti-antisense mechanism, independent of its action as an antisilencer of transcription'
---
authors: 'Majdalani, Nadim and Hernandez, David and Gottesman, Susan'
title: 'Regulation and mode of action of the second small RNA activator of RpoS translation, RprA'
---
authors: 'Majdalani, Nadim'
doi: 10.1080/10409230590918702
title: Bacterial Small RNA Regulators
---
authors: 'Maki, Kimika and Uno, Kanako and Morita, Teppei and Aiba, Hiroji'
doi: 10.1073/pnas.0803106105
title: 'RNA, but not protein partners, is directly responsible for translational silencing by a bacterial Hfq-binding small RNA'
---
authors: 'Maniatis, Tom and Reed, Robin'
title: An extensive network of coupling among gene expression machines
---
authors: 'Maniatis, Tom and Tasic, Bosiljka'
title: Alternative pre-mRNA splicing and proteome expansion in metazoans
---
authors: 'Mann, Martin and Backofen, Rolf and Will, Sebastian'
title: Equivalence Classes of Optimal Structures in HP Protein Models Including Side Chains
---
authors: 'Mann, Martin and Tack, Guido and Will, Sebastian'
title: Decomposition During Search
---
arxivid: 0712.2389v1
authors: 'Mann, Martin and Tack, Guido and Will, Sebastian'
title: Decomposition During Search for Propagation-Based Constraint Solvers
---
authors: 'Mann, Martin and Will, Sebastian and Backofen, Rolf'
title: 'HPdesign:, Inverse, Folding, of, Proteins'
---
authors: 'Mann, Martin and Maticzka, Daniel and Saunders, Rhodri and Backofen, Rolf'
title: Classifying protein-like sequences in arbitrary lattice protein models using LatPack
---
authors: 'Manning, Gerard and Plowman, Gregory D. and Hunter, Tony and Sudarsanam, Sucha'
title: Evolution of protein kinase signaling from yeast to man
---
authors: 'Mao, Yuan and Najafabadi, Hamed Shateri and Salavati, Reza'
doi: 10.1186/1471-2164-10-355
title: Genome-wide computational identification of functional RNA elements in Trypanosoma brucei
---
authors: '勇), Mao Yong (毛 and (周晓波), Zhou Xiao-bo and (皮道映), PI Dao-ying and (孙优贤), Sun You-xian and C., Wong Stephen T.'
doi: 10.1631/jzus.2005.B0961
title: Parameters selection in gene selection using Gaussian kernel support vector machines by genetic algorithm
---
authors: 'Maquat, Lynne E.'
doi: 10.1038/nrm1310
title: 'NONSENSE-MEDIATED mRNA DECAY: SPLICING, TRANSLATION AND mRNP DYNAMICS'
---
authors: 'Marcotte, Edward M'
title: 'Computational genetics: finding protein function by nonhomology methods'
---
authors: 'Mardis, Elaine R.'
doi: 10.1016/j.tig.2007.12.007
title: The impact of next-generation sequencing technology on genetics
---
authors: 'Margolin, Adam A and Nemenman, Ilya and Basso, Katia and Wiggins, Chris and Stolovitzky, Gustavo and Favera, Riccardo Dalla and Califano, Andrea'
doi: 10.1186/1471-2105-7-S1-S7
title: 'ARACNE: An Algorithm for the Reconstruction of Gene Regulatory Networks in a Mammalian Cellular Context'
---
authors: 'Marilley, M. and Radebaugh, C. A. and Geiss, G. K. and Laybourn, P. J. and Paule, M. R.'
doi: 10.1007/s00438-002-0708-z
title: DNA structural variation affects complex formation and promoter melting in ribosomal RNA transcription
---
authors: 'Maritan, Amos and Micheletti, Cristian and Trovato, Antonio and Banavar, Jayanth R.'
title: Optimal shapes of compact strings
---
authors: 'Markstein, Michele and Markstein, Peter and Markstein, Vicky and Levine, Michael S.'
doi: 10.1073/pnas.012591199
title: Genome-wide analysis of clustered Dorsal binding sites identifies putative target genes in the Drosophila embryo
---
authors: 'Bauer, Markus and Klau, Gunnar W. and Reinert, Knut'
title: Accurate Multiple Sequence-Structure Alignment of RNA Sequences Using Combinatorial Optimization
---
authors: 'Markwick, Phineus R. L. and Malliavin, Therese and Nilges, Michael'
doi: 10.1371/journal.pcbi.1000168
title: 'Structural Biology by NMR: Structure, Dynamics, and Interactions'
---
authors: 'Marson, Alexander and Levine, Stuart S. and Cole, Megan F. and Frampton, Garrett M. and Brambrink, Tobias and Johnstone, Sarah and Guenther, Matthew G. and Johnston, Wendy K. and Wernig, Marius and Newman, Jamie and Mauro Calabrese, J. and Dennis, Lucas M. and Volkert, Thomas L. and Gupta, Sumeet and Love, Jennifer and Hannett, Nancy and Sharp, Phillip A. and Bartel, David P. and Jaenisch, Rudolf and Young, Richard A.'
doi: 10.1016/j.cell.2008.07.020
title: Connecting microRNA Genes to the Core Transcriptional Regulatory Circuitry of Embryonic Stem Cells
---
authors: 'Martin, James F. and Miano, Joseph M. and Hustad, Carolyn M. and Copeland, Neal G. and Jenkins, Nancy A. and Olson, Eric N.'
title: A Mef2 Gene That Generates a Muscle-Specific Isoform via Alternative mRNA Splicing
---
authors: 'MARTIN III, Glover W. and Harney, John W. and Berry, Marla J.'
title: Functionality of mutations at conserved nucleotides in eukaryotic SECIS elements is determined by the identity of a single nonconserved nucleotide
---
authors: 'Martineau, Yvan and Le Bec, Christine and Monbrun, Laurent and Allo, Valerie and Chiu, Ing-Ming and Danos, Olivier and Moine, Herve and Prats, Herve and Prats, Anne-Catherine'
doi: 10.1128/MCB.24.17.7622-7635.2004
title: Internal Ribosome Entry Site Structural Motifs Conserved among Mammalian Fibroblast Growth Factor 1 Alternatively Spliced mRNAs
---
authors: 'Martinez, Hugo M.'
title: An RNA folding rule
---
authors: 'Marzi, Stefano and Myasnikov, Alexander G. and Serganov, Alexander and Ehresmann, Chantal and Romby, Pascale and Yusupov, Marat and Klaholz, Bruno P.'
doi: 10.1016/j.cell.2007.07.008
title: Structured mRNAs Regulate Translation Initiation by Binding to the Platform of the Ribosome
---
authors: 'Mason, Jody M. and Schmitz, Mark A. and Muller, Kristian M. and Arndt, Katja M.'
doi: 10.1073/pnas.0509880103
title: 'Semirational design of Jun-Fos coiled coils with increased affinity: Universal implications for leucine zipper prediction and design'
---
authors: 'Masse, Eric and Gottesman, Susan'
doi: 10.1073/pnas.032066599
title: A small RNA regulates the expression of genes involved in iron metabolism in Escherichia coli
---
authors: 'Masse, Eric and Vanderpool, Carin K. and Gottesman, Susan'
doi: 10.1128/JB.187.20.6962-6971.2005
title: Effect of RyhB Small RNA on Global Iron Use in Escherichia coli
---
authors: 'Mathe, Catherine and Sagot, Marie-France and Schiex, Thomas and Rouze, Pierre'
title: 'Current methods of gene prediction, their strengths and weaknesses'
---
authors: 'Mathews, David H. and Burkard, Mark E. and Freier, Susan M. and Wyatt, Jacqueline R. and Turner, Douglas H.'
title: Predicting oligonucleotide affinity to nucleic acid targets
---
authors: 'Mathews, David H. and Disney, Matthew D. and Childs, Jessica L. and Schroeder, Susan J. and Zuker, Michael and Turner, Douglas H.'
doi: 10.1073/pnas.0401799101
title: Incorporating chemical modification constraints into a dynamic programming algorithm for prediction of RNA secondary structure
---
authors: 'Mathews, David H. and Turner, Douglas H.'
doi: 10.1006/jmbi.2001.5351
title: 'Dynalign: An Algorithm for Finding the Secondary Structure Common to Two RNA Sequences'
---
authors: 'Mathews, David H. and Sabina, Jeffrey and Zuker, Michael and Turner, Douglas H.'
title: Expanded Sequence Dependence of Thermodynamic Parameters Improves Prediction of RNA Secondary Structure
---
authors: 'Matlin, Arianne J. and Clark, Francis and Smith, Christopher W. J.'
doi: 10.1038/nrm1645
title: 'UNDERSTANDING ALTERNATIVE SPLICING: TOWARDS A CELLULAR CODE'
---
authors: 'Matos, Paulo and Collard, John G. and Jordan, Peter'
doi: 10.1074/jbc.M308215200
title: Tumor-related Alternatively Spliced Rac1b Is Not Regulated by Rho-GDP Dissociation Inhibitors and Exhibits Selective Downstream Signaling
---
authors: 'Mattick, John S.'
doi: 10.1002/bies.10332
title: 'Challenging the dogma: the hidden layer of non-protein-coding RNAs in complex organisms'
---
authors: 'Mattick, John S'
title: 'Introns: evolution and function'
